sqlalchemy-stubs/dialects/mssql/base.pyi

```diff
@@ -0,0 +1,62 @@
+from typing import Any, Optional
+from ... import types as sqltypes
+from ...types import INTEGER as INTEGER, BIGINT as BIGINT, SMALLINT as SMALLINT, DECIMAL as DECIMAL, NUMERIC as NUMERIC, \
+    FLOAT as FLOAT, DATETIME as DATETIME, DATE as DATE, BINARY as BINARY, \
+    TEXT as TEXT, VARCHAR as VARCHAR, NVARCHAR as NVARCHAR, CHAR as CHAR, NCHAR as NCHAR
+
+class REAL(sqltypes.REAL):
+    __visit_name__: str = ...
+
+class TINYINT(sqltypes.Integer):
+    __visit_name__: str = ...
+
+class TIME(sqltypes.TIME):
+    __visit_name__: str = ...
+
+class _DateTimeBase(object): ...
+
+class SMALLDATETIME(_DateTimeBase, sqltypes.DateTime):
+    __visit_name__: str = ...
+
+class DATETIME2(_DateTimeBase, sqltypes.DateTime):
+    __visit_name__: str = ...
+
+class DATETIMEOFFSET(sqltypes.TypeEngine):
```

Comment:
I prefer to always have explicit type arguments for generics (even if it is `Any`). For example, here it is better to write `TypeEngine[Any]`.

Reply:
I'll update those. Thanks!